packages/world/ts/worldEvents.test.ts

```diff
@@ -1,7 +1,7 @@
 import { describe, expect, it } from "vitest";
 import IWorldFactoryAbi from "../out/IWorldFactory.sol/IWorldFactory.abi.json";
 import IBaseWorldAbi from "../out/IBaseWorld.sol/IBaseWorld.abi.json";
-import { helloWorldEvent, worldDeployedEvent } from "./worldEvents";
+import { helloWorldEvent, worldDeployedEvent } from "./worldEvents.js";
```

Comment:
I realize we wanna move towards this but is the `.js` extension here needed?

Reply:
I can get rid of this, it was likely just an accident I was testing out some different settings to get json resolution working